ecs-proxies-deploy-sandbox.yml

```diff
@@ -14,11 +14,11 @@ docker_service:
         value: "STRICT"
       - name: SDS_SPINE_CORE_ODS_CODE
         value: "YES"
-      - name: CPM_CLIENT_KEY
-        value: JrxvR6WsyTEu8BVVA1qhTbKqoktoVn0y
       - name: CPM_PATH_URL
         value: cpm-qa-sandbox
     secrets:
+      - name: CPM_CLIENT_KEY
+        valueFrom: "arn:aws:secretsmanager:eu-west-2:{{ PTL_ACCOUNT_ID }}:secret:ptl/spine-directory-service/cpm-apikey"
```

Comment:
This particular file is confusing me - isnt sandbox supposed to be int? and why is internal-qa sandbox underneath in the APIGEE_URL section when we already have an internal-qa-sanbox yaml? 🤯